### PR TITLE
update owslib to 0.19.2

### PR DESCRIPTION
**Pull request by davidcaron**

### CODE REVIEW

**Comment by davidcaron** [comment]
@fmigneault any minor version release planned soon?

**Comment by fmigneault** [comment]
I was waiting for the package (#78)
Can be udpated soon, but need to fix the requirements. 
Python 3.5 fails because owslib dropped it.

**Comment by davidcaron** [comment]
If owslib dropped support for python 3.5, you're right that this would mean weaver should to when we update to this version.

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,6 @@
       classifiers=[
           "Natural Language :: English",
           "Programming Language :: Python :: 2.7",
-          "Programming Language :: Python :: 3.5",
```
Comment (fmigneault) [inline setup.py]: also update the below `python_requires`

Comment (davidcaron) [reply setup.py]: done

**Pull request merged**